include/pokemon.h

```diff
@@ -330,10 +338,69 @@ struct SpeciesInfo /*0x24*/
  /* 0x16 */ u8 eggGroups[2];
  /* 0x18 */ u16 abilities[NUM_ABILITY_SLOTS]; // 3 abilities, no longer u8 because we have over 255 abilities now.
  /* 0x1E */ u8 safariZoneFleeRate;
- /* 0x1F */ u8 bodyColor : 7;
```

Comment:
Non blocking, but given how much data is in this struct, I really think it would be a good idea to create some substructs to group together logical data sets e.g. BaseStats, DexInfo, SpriteInfo etc.